SBR.md

```diff
@@ -460,7 +461,15 @@ The Random Value SHALL be unique in each email. The Random Value SHALL remain va
 
 The Random Value SHALL be reset upon each instance of the email sent by the CA and, if intended for additional use as an authentication factor, upon first use.  
 
-#### 3.2.2.3 CAA Records
+#### 3.2.2.3 Validating applicant as operator of associated mail server(s)
+
+Confirming the Applicant's control over the `rfc822Name` or `otherName` of type `id-on-SmtpUTF8Mailbox` email address by confirming control of the SMTP FQDN to which a message delivered to the email address should be directed. The SMTP FQDN MUST be identified using the address resolution algorithm defined in RFC 5321 Section 5.1 which determines which SMTP FQDNs are authoritative for a given email address. If more than one SMTP FQDNs have been discovered, the CA MUST verify control of an SMTP FQDN following the selection process at RFC 5321 Section 5.1. Aliases in MX record RDATA MUST NOT be used for this validation method.
+
+When confirming the Applicant's control of the SMTP FQDN, the CA MUST use the methods in Section 3.2.2.4 of the Baseline Requirements for the Issuance and Management of Publicly‐Trusted Certificates.
```

Comment:
For the sake of clarity, here the text deals with a `primary FQDN` (i.e., example.com from the email address jimmyg@example.com) as well as a `target SMTP FQDN` that is indicated in the MX record for example.com (i.e., ASPMX.L.GOOGLE.COM or example-com.mail.protection.outlook.com).
In this paragraph I assume you are referring to the Applicant as being the email service provider who can be validated using TLS BR 3.2.2.4 as controlling the `target SMTP FQDN`.

Reply:
Within the context of this validation method, the only FQDN that is checked is the SMTP FQDN. You are right that in this case the Applicant is the email service provider. Of course, if an entity owns example.com and example.org with a single SMTP FQDN (e.g. mail.example.com), then the entity itself acts as an email service provider.